demo/src/pages/interApplicationBusSend/InterApplicationBusSend.tsx

```diff
@@ -9,28 +9,53 @@ const TOPIC = "demo-topic";
 
 const codeExample = `import {useInterApplicationBusSend, useInterApplicationBusSubscribe} from "openfin-react-hooks";
```

Comment:
Do the examples for each of the hooks need updating to match the new interface?

Reply:
Yes, they have been changed (unless I've made a mistake!). It's not very clear in the PR, but the code example ends on the (new) line 45 of this file.